packages/contract/src/test-utils/launch-test-node.ts

```diff
@@ -130,7 +130,7 @@ function getWalletForDeployment(config: DeployContractConfig, wallets: WalletUnl
   return wallets[config.walletIndex];
 }
 
-export async function launchTestNode<TFactories extends DeployContractConfig[]>({
+export async function launchTestNode<const TFactories extends DeployContractConfig[]>({
```

Comment:
All of this because of a missing `const`? 🙃

Reply:
Yeah, a standard array of multiple factories `[F1, F2, F3, F1]` is inferred as `(F1 | F2 | F3)[]`, whereas a `const` array is inferred as `[F1, F2, F3, F1]`.